### PR TITLE
Che 212  disable field auto completion after its adjustments

### PR DESCRIPTION
**Pull request by olexii4**
@ashumilova WDYT?

### CODE REVIEW

**Comment by benoitf** [comment]
I need to test first

**Comment by ashumilova** [comment]
tested - it's working : +1

File: dashboard/src/app/projects/create-project/create-project.controller.js [hunk 1]
```diff
@@ -1117,4 +1066,48 @@ export class CreateProjectCtrl {
...
+      this.workspaceName = angular.copy(name);
+    }
+  }
+
+  /**
+   * Set project name
+   * @param name
+   */
+  setProjectName(name) {
+    if(!name) {
```
Comment (benoitf) [inline dashboard/src/app/projects/create-project/create-project.controller.js]: missing space after if (

File: dashboard/src/app/projects/create-project/create-project.controller.js [hunk 2]
```diff
@@ -1117,4 +1066,48 @@ export class CreateProjectCtrl {
...
+      return;
+    }
+    if(!this.defaultProjectName || this.defaultProjectName === this.projectName){
```
Comment (benoitf) [inline dashboard/src/app/projects/create-project/create-project.controller.js]: spaces around parenthesis

File: dashboard/src/app/projects/create-project/create-project.controller.js [hunk 3]
```diff
@@ -1117,4 +1066,48 @@ export class CreateProjectCtrl {
...
+    }
+    this.importProjectData.project.name = this.projectName;
+  }
+
+  /**
+   * Set project description
+   * @param description
+   */
+  setProjectDescription(description) {
+    if(!description) {
```
Comment (benoitf) [inline dashboard/src/app/projects/create-project/create-project.controller.js]: spaces

File: dashboard/src/app/projects/create-project/create-project.controller.js [hunk 4]
```diff
@@ -1117,4 +1066,48 @@ export class CreateProjectCtrl {
...
+      return;
+    }
+    if(!this.defaultProjectDescription || this.defaultProjectDescription === this.projectDescription){
```
Comment (benoitf) [inline dashboard/src/app/projects/create-project/create-project.controller.js]: spaces

File: dashboard/src/app/workspaces/create-workspace/create-workspace.controller.js [hunk 5]
```diff
@@ -74,10 +75,24 @@ export class CreateWorkspaceCtrl {
     this.stack = stack;
     this.recipeUrl = null;
     this.isCustomStack = false;
-    if (stack && stack.workspaceConfig && stack.workspaceConfig.name) {
-      this.workspaceName = angular.copy(stack.workspaceConfig.name);
-    } else {
-      this.generateWorkspaceName();
+    if(stack.workspaceConfig && stack.workspaceConfig.name){
```
Comment (benoitf) [inline dashboard/src/app/workspaces/create-workspace/create-workspace.controller.js]: spaces

File: dashboard/src/app/workspaces/create-workspace/create-workspace.controller.js [hunk 6]
```diff
@@ -74,10 +75,24 @@ export class CreateWorkspaceCtrl {
...
+    }
+    this.generateWorkspaceName();
+  }
+
+  /**
+   * Set workspace name
+   * @param name
+   */
+  setWorkspaceName(name) {
+    if(!name) {
```
Comment (benoitf) [inline dashboard/src/app/workspaces/create-workspace/create-workspace.controller.js]: spaces

File: dashboard/src/app/workspaces/create-workspace/create-workspace.controller.js [hunk 7]
```diff
@@ -74,10 +75,24 @@ export class CreateWorkspaceCtrl {
...
+      return;
+    }
+    if(!this.defaultWorkspaceName || this.defaultWorkspaceName === this.workspaceName){
```
Comment (benoitf) [inline dashboard/src/app/workspaces/create-workspace/create-workspace.controller.js]: spaces

**Comment by benoitf** [comment]
works fine good job +1 (if minors code style issues are fixed)

**Pull request merged**